src/libstd/keyword_docs.rs

```diff
@@ -59,21 +395,108 @@ mod let_keyword { }
 
 #[doc(keyword = "struct")]
 //
-/// The `struct` keyword.
+/// The keyword used to define structs.
```

Comment:
Since you mentioned ADTs, it would be nice to mention that `struct`s are product types.